### DISCUSSION TITLE
Need to use @antv/g6 for my project

### DISCUSSION THREAD

**Discussion by pgoel-ent** [antvis/G6#3964] [Q&A] [answered]
We are using "@antv/g6": "^4.3.0", and it have one dependency - @webgpu/glslang , which is Proprietary module, so what should we do to not use this paid plugin, is there any stable version available which is not using "@webgpu/glslang"

**Comment by xiaoiver** [comment, marked as answer]
We don't use this dependency any more. You can check its dependents here:
https://www.npmjs.com/package/@webgpu/glslang?activeTab=dependents
